serving/docs/lmi/deployment_guide/configurations.md

```diff
@@ -125,17 +125,20 @@ You can find these configurations in the respective [user guides](../user_guides
 
 All LMI Configuration keys available in the `serving.properties` format can be specified as environment variables.
 
-The translation for `engine` is unique. The configuration `engine=<engine>` is translated to `SERVING_LOAD_MODELS=test::<engine>=/opt/ml/model`.
-For example:
+The property `option.model_id` is unique. It is translated to `HF_MODEL_ID`.
 
-* `engine=Python` is translated to environment variable `SERVING_LOAD_MODELS=test::Python=/opt/ml/model`
-* `engine=MPI` is translated to environment variable `SERVING_LOAD_MODELS=test::MPI=/opt/ml/model`
+The property `engine` is translated to `OPTION_ENGINE`.
+By default, LMI will use the Python engine. You can use `OPTION_ENGINE=Python` to explicitly set the engine.
+To use the MPI engine, you should also provide `OPTION_MPI_MODE=true`.
```

Comment:
Avoid use "MPI engine" here,

If your engine requires use mpi launcher, you should also provide ...